docs/FrameDescriptions.md

```diff
@@ -1,21 +1,121 @@
+) provides us the rotation from [inertial frame](#inertial-frame) (attached to the human head at the begning) to the oculus headset at timestep `t` expressed in preferred coordinate system:
+
+```
+iDynTree::toEigen(m_oculusRoot_T_headOculus).block(0, 0, 3, 3)
+                = iDynTree::toEigen(iDynTree::Rotation::RPY(-desiredHeadOrientationVector(1),
+                                                            desiredHeadOrientationVector(0),
+                                                            desiredHeadOrientationVector(2)));
+```                                                           
 
 <p align="center">
   <img width="500" height="500" src="./images/Oculus.png">
 </p>
 
-**Virtualizer (treadmill)**
+## treadmill (Virtualizer Cyberith)
+
+According to the measurement information we acquire from the cyberith treadmill, we identify the frame of Virtualizer as following:
 
 <p align="center">
-  <img width="600" height="300" src="./images/Virtualizer%2Bothers.jpg">
+  <img width="300" src="./images/virtualizerFrame.jpg">
 </p>
 
-**Xsens Suit**
+Threfore [we have:](https://github.com/robotology/walking-teleoperation/blob/60b449e6e8d5120a2a11ca2997521f46c51821c1/modules/Oculus_module/src/HeadRetargeting.cpp#L105)
+
+```
+m_oculusInertial_R_teleopFrame = iDynTree::Rotation::RotZ(-playerOrientation);
+```
+The rotation of the operator (called player here) inside the treadmill is defined with `playerOrientation`, and the rotation of it is expressed in preferred coordinates of the `I_treadmill`. In order to express the rotation in [inertial coordinates](#inertial-frame) (in the code is mentioned by `oculusInertial`), we perform the following transformation
+```
+ rotz(- playerOrientation)= rotx(pi) * rotz(playerOrientation) * rotx(-pi)
+```
+When the human rotates inside the virtualizer, not only he has rotation around `z` axis, but also the _teleoperaition frame_  translates as following figure shows:
+
 
 <p align="center">
-  <img width="500" height="500" src="./images/humanXses.png">
-</p> | <p align="center">
-  <img width="500" height="500" src="./images/RobotFrames.png">
+  <img width="
+  <img width="300" src="./images/teleoperationFrameTransformation.jpg">
+</p>
+
+To find the translation, we assume when the human rotates, he is looking forward, and we compute the translation using the acquired oculus headset data.
+
+
+
+# Hand Frames
+
+The human and robot hand frames are defined similarly,i.e.,
+
+<p align="center">
+  <img width="300" src="./images/RobotHand.jpeg">
 </p> 
+
+What we are interested in the teleoperation scnenario is the transfromation `m_teleopRobotFrame_T_handRobotFrame`, i.e., [the transformation from the rorobt teleoperation frame to the robot hand frames](https://github.com/robotology/walking-teleoperation/blob/60b449e6e8d5120a2a11ca2997521f46c51821c1/modules/Oculus_module/src/HandRetargeting.cpp#L74-L77)
```

Comment:
Some typos:
- "scnenario"
- "transfromation"
- "rorobt"

Reply:
All the text has been proofread and typos are fixed.